Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Grid-Editor/index.md

```diff
@@ -0,0 +1,625 @@
+
+Once you have added an Element Type as a Block Type on your Block Grid Data Type you have the option to configure it.
+
+![Block List - Data Type Block Configuration](images/BlockListEditor_DataType_Blocks.png)
+> TODO: screenshot
+
+Each Block has a set of properties that are optional to configure. These are described below.
+
+### Block Appearance
+
+Customize the user experience for your content editors when they work with the blocks in the Content section.
+
+- **Label** - Define a label for the appearance of the Block in the editor. The label can use AngularJS template-string-syntax to display values of properties.
+  :::tip
+  Label example: "My Block {{myPropertyAlias}}" will be shown as: "My Block FooBar". 
+  
+  You can also use more advanced expression using AngularJS filters, like `{{myPropertyAlias | limitTo:100}}` or for a property using Richtext editor `{{myPropertyAlias | ncRichText | truncate:true:100}}`. It is also possible to use properties from the Settings model by using `{{$settings.propertyAlias}}`.
+  :::
+- **Custom view** - Overwrite the AngularJS view for the block presentation in the Content editor. Use this to make a more visual presentation of the block or make your own editing experience by adding your own AngularJS controller to the view.
+- **Custom stylesheet** - Pick your own stylesheet to be used for this block in the Content editor. By adding a stylesheet the styling of this block will become scoped. This means that the default backoffice styles are no longer present for the view of this block.
+- **Overlay editor size** - Set the size for the Content editor overlay for editing this block.
+
+### Catalogue appearance
+
+These properties refer to how the Block is presented in the Block catalogue when editors choose which Blocks to use for their content.
+
+- **Background color** - Define a background color to be displayed beneath the icon or thumbnail. Example: `#424242`.
+- **Icon color** - Change the color of the Element Type icon. Example: `#242424`.
+- **Thumbnail** - Pick an image or SVG file to replace the icon of this Block in the catalogue.
+
+The thumbnails for the catalogue are presented in the format of 16:10, and we recommend a resolution of 400px width and 250px height.
+
+### Layout options
+
+**TO BE DONE**
+
+### Data Models
+
+It is possible to use two separate Element Types for your Block Types. It is required to have one for Content and optional to add one for Settings.
+
+- **Content model** - This presents the Element Type used as model for the Content section of this Block. This cannot be changed, but you can open the Element Type to perform edits or view the properties available. Useful when writing your Label.
+- **Settings model** - Add a Settings section to your Block based on a given Element Type. When selected you can open the Element Type or choose to remove the Settings section again.
+
+### Areas
+
+**TO BE DONE**
```

Comment:
Will you be writing this, or is this something we needs to help complete?

Reply:
This will have to be updated as UI is changed.
It could be done by you when you take screenshots?

Reply:
Ive added some about this, also some comments for what to write more about.